src/js/gi/utils/helpers.js

```diff
@@ -7,3 +7,7 @@ export function getCurrency(value) {
 
   return isNumeric(amount) ? +amount : 0;
 }
+
+export function formatCurrency(value) {
+  return `$${Math.round(value)}`;
```

Comment:
this should be ok. I think we miss out on the automatic comma separators but it's not a big deal. I also think it lacks support in a few mobile browsers, notably Android.